src/App.tsx

```diff
@@ -15,48 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface SortingParams {
+  sortField: string;
+  reversed: boolean;
+}
+
+enum SortType {
+  alphabet = 'alphabet',
+  length = 'length',
+}
```

Comment:
Enum's elements should be CamelCase or UPPER_CASE
```suggestion
enum SortType {
  Alphabet = 'alphabet',
  Length = 'length',
}
```